addons/binding/org.openhab.binding.amazonechocontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,425 @@
+            <channel id="bluetoothDeviceName" typeId="bluetoothDeviceName" />
+            <channel id="radioStationId" typeId="radioStationId" />
+            <channel id="radio" typeId="radio" />
+            <channel id="amazonMusicTrackId" typeId="amazonMusicTrackId" />
+            <channel id="amazonMusicPlayListId" typeId="amazonMusicPlayListId" />
+            <channel id="amazonMusicPlayListIdLastUsed" typeId="amazonMusicPlayListIdLastUsed" />
+            <channel id="amazonMusic" typeId="amazonMusic" />
+            <channel id="remind" typeId="remind" />
+            <channel id="playAlarmSound" typeId="playAlarmSound" />
+            <channel id="playFlashBriefing" typeId="playFlashBriefing" />
+            <channel id="playTrafficNews" typeId="playTrafficNews" />
+            <channel id="playWeatherReport" typeId="playWeatherReport" />
+            <channel id="playGoodMorning" typeId="playGoodMorning" />
+            <channel id="startRoutine" typeId="startRoutine" />
+            <channel id="playMusicProvider" typeId="playMusicProvider" />
+            <channel id="playMusicVoiceCommand" typeId="playMusicVoiceCommand" />
+        </channels>
+        <representation-property>serialNumber</representation-property>
+        <config-description>
+            <parameter name="serialNumber" type="text" required="true">
+                <label>Serial Number</label>
+                <description>You will find the serial number of your device in the Alexa app</description>
+            </parameter>
+        </config-description>
+    </thing-type>
+    <thing-type id="flashbriefingprofile">
+        <supported-bridge-type-refs>
+            <bridge-type-ref id="account" />
+        </supported-bridge-type-refs>
+        <label>Flash Briefing Profile</label>
+        <description>Store and load a flash briefing configuration</description>
+        <channels>
+            <channel id="save" typeId="save" />
+            <channel id="active" typeId="active" />
+            <channel id="playOnDevice" typeId="playOnDevice" />
+        </channels>
+    </thing-type>
+    <channel-type id="save">
+        <item-type>Switch</item-type>
+        <label>Save</label>
+        <description>Save the current flash briefing configuration (Write only)</description>
+    </channel-type>
+    <channel-type id="active">
+        <item-type>Switch</item-type>
+        <label>Active</label>
+        <description>Activate this flash briefing configuration</description>
+    </channel-type>
+    <channel-type id="playOnDevice">
+        <item-type>String</item-type>
+        <label>Play On Device</label>
+        <description>Plays the briefing on the device (serial number or name, write only)</description>
+    </channel-type>
+    <channel-type id="switch">
+        <item-type>Switch</item-type>
+        <label>Switch</label>
+        <description>Turns the device on or off</description>
+    </channel-type>
+    <channel-type id="dimmer">

```

Comment:
Sorry, this is a mistake, I have forgotten to remove it. It's a remnant of the smarthome feature which I have moved to a other branch because it does not work for the most users.